src/App.css

```diff
@@ -36,3 +36,38 @@
     transform: rotate(360deg);
   }
 }
+
+.buttonPanel {
+  width: 720px;
```

Comment:
Let's keep to the project requirement:
I think the width should be `700px`
![image](https://user-images.githubusercontent.com/31897434/87582144-e1624600-c6d1-11ea-99b8-af83dfe0b262.png)